install/docker/debian-docker-install.md

```diff
@@ -0,0 +1,121 @@
+        
+        ```
+        # single runtime
+        DOCKER_OPTS="-D --add-runtime kata-runtime=/usr/bin/kata-runtime --default-runtime=kata-runtime
+        # multiple runtime options (kata default)
+        #DOCKER_OPTS="-D --add-runtime kata-runtime=/usr/bin/kata-runtime cc-runtime=/usr/bin/cc-runtime --default-runtime=kata-runtime
+        ```
+        > Question: When changing the default runtime, does docker remember which runtime a container was previously created with?
+        > Answer: It looks so (probably stored in container json), also  note the [support for specifying `runtime: `](https://github.com/docker/compose/pull/5405) in docker-compose (v3)
+        
+        > **Note:** other options can be added to the `DOCKER_OPTS` in `/etc/default/docker` either as separate commands or as a unified command,
+        > for instance, the following is valid syntax:
+        
+        ```
+        DOCKER_OPTS="-D --add-runtime kata-runtime=/usr/bin/kata-runtime --default-runtime=kata-runtime \
+        --storage-driver devicemapper \
+        --storage-opt dm.basesize=40G \
+        --storage-opt dm.thinpooldev=/dev/mapper/docker--vg-docker--pool \
+        --storage-opt dm.fs=ext4"
+        # local docker registry setting
+        DOCKER_OPTS="${DOCKER_OPTS} --registry-mirror=http://docker-registry:5000"
+        ```
+    
+    
+    2. systemd
+
+        ```bash
+        $ sudo mkdir -p /etc/systemd/system/docker.service.d/
+        $ cat <<EOF | sudo tee /etc/systemd/system/docker.service.d/kata-containers.conf
+        [Service]
+        ExecStart=
+        ExecStart=/usr/bin/dockerd -D --add-runtime kata-runtime=/usr/bin/kata-runtime --default-runtime=kata-runtime
+        EOF
+        ```
+
+    3. systemd Docker `daemon.json`
+
+        Add the following definitions to `/etc/docker/daemon.json`:
+
+        ```json
+        {
+          "default-runtime": "kata-runtime",
+          "runtimes": {
+            "kata-runtime": {
+              "path": "/usr/bin/kata-runtime"
+            }
+          }
+        }
+        ```
+
+3. Restart the Docker systemd service with one of the following (depending on init choice):
+
+  1. sysVinit
+  
+    ```bash
+    $ sudo /etc/init.d/docker stop
+    $ sudo /etc/init.d/docker start
+    ```
+
+    to watch for errors:
+
+    ```bash
+    tail -f /var/log/docker.log
+    ```
+    
+  2. systemd
+
+   ```bash
+   $ sudo systemctl daemon-reload
+   $ sudo systemctl restart docker
+   ```
+
+4. Run Kata Containers
+
+   You are now ready to run Kata Containers:
+
+   ```bash
+   $ sudo docker run busybox uname -a
+   ```
+
+   The previous command shows details of the kernel version running inside the
+   container, which is different to the host kernel version.
+
+   > **Note**: You can also specify the runtime here to check kata vs clear containers (again you'll find different kernel versions)
```

Comment:
Again, I'm not convinced this is worth keeping here. This is a docker concern and if we do want to document this, it should be done once for all the distros we offer packages for and not in a Debian specific document.